plugin_ide.core.tests/src-lang/melnorme/lang/ide/core/engine/StructureModelTest.java

```diff
@@ -60,8 +60,7 @@ public FixtureSourceModelManager() {
 			super(new DocumentReconcileManager(), new ProblemMarkerUpdater());
 		}
 		
-		public final SourceFileStructure DEFAULT_STRUCTURE = new SourceFileStructure(null, null, 
-			(Indexable<ParserError>) null);
+		public final SourceFileStructure DEFAULT_STRUCTURE = new SourceFileStructure(null, (Indexable<ParserError>) null);
```

Comment:
You're right. There is no issue. My bad!
I merged your changes and restored the `StructureElement`s to their original shape.